Couler Example.ipynb

```diff
@@ -55,7 +57,7 @@
    "outputs": [],
    "source": [
     "!pip config --user set global.index-url https://jfrog.aaw.cloud.statcan.ca/artifactory/api/pypi/pypi-remote/simple\n",
-    "!python3 -m pip install git+https://github.com/couler-proj/couler --ignore-installed"
```

Comment:
--ignore-install can create dangling files/packages in installation directories and should be used by advanced users only (ones that are aware they may make a mess). I've replaced it with a safer option but commented that out as well since it shouldn't be needed.

Reply:
Good to know! Thanks!